### ISSUE TITLE
Add `dbt source freshness` results into Test Results and Test Runs tabs in report UI

### ISSUE THREAD

**Issue by ivan-toriya** [elementary-data/elementary#450]
**Is your feature request related to a problem? Please describe.**
I want to see the overview and history of `dbt source freshness` results

**Describe the solution you'd like**
`dbt source freshness` results listed in Test Results and Test Runs tabs in report UI

**Describe alternatives you've considered**
Only have them as alerts in Slack. (current implementation)

**Additional context**
None

**Would you be willing to contribute this feature?**
Yes

**Comment by elongl** [comment]
We can begin working on this again now that there's `edr run-operation upload-source-freshness`.

**Comment by christopherekfeldt** [comment]
Yes would be great if the "Freshness" GUI in the report would display the results of the "dbt source freshness" runs as well as the elementary.anomalies.freshness tests.

**Comment by benoit-tropos-io** [comment]
Yes would be super great. We already have 3 huge projects that are desperately asking for this feature.

**Comment by shyamk136** [comment]
+1 on this feature. The Freshness Anomaly test is great for viewing trends over time but a simple 'freshness' result based on the timing helps the report viewer understand if there is any immediate issue that needs action.

**Comment by fredmny** [comment]
Just +1'ing this. It would be really awesome.

**Comment by harukitake** [comment]
+1 I really hope this feature.

**Comment by viniciusnunest** [comment]
+1 .

**Comment by paritoshg2010** [comment]
+1, Looking forward to this feature :)

**Comment by Maayan-s** [comment]
Hey all, 
We are finally implementing this! 
This is the current design, would be great to get your feedback on it before we start developing: 

https://www.figma.com/file/OX8pEefdDd2zBTGhzpzywl/Source-freshness-visualization-Pr[…]tary)?type=design&node-id=0-1&mode=design&t=as6eoYaEfc71CIXU-0

@ivan-toriya @paritoshg2010 @viniciusnunest @harukitake @fredmny @shyamk136 @benoit-tropos-io @christopherekfeldt

**Comment by viniciusnunest** [comment]
@Maayan-s  Your design for indicating source freshness on the test page is excellent. Will similar features be incorporated into the main dashboard as well?

**Comment by Maayan-s** [comment]
@viniciusnunest thanks! 
First step is to add the results to the freshness tile in the dashboard:
<img width="226" alt="image" src="https://github.com/elementary-data/elementary/assets/22072028/d1e21a31-8c83-4754-9abb-4dc1ef69bc1b">

What kind of indicator did you have in mind?

**Comment by viniciusnunest** [comment]
Just that and the history at the bottom of the dashboard as we have tests would be very good, as this is something related to the source, this distinction would be very nice to give visibility

Like this : 
<img width="362" alt="image" src="https://github.com/elementary-data/elementary/assets/104783995/fb07cbe8-af7b-4ff1-b7a0-2f73ce2b3162">

**Comment by fredmny** [comment]
@Maayan-s Coming a little late since I was on vacation, but I quietly celebrated when I saw the the notification for your message on my phone as soon as you posted it.

I really like the design. It's way better than I expected! I would +1 the request for a KPI-like indicator on the main dashboard, like the image you sent afterwards. And adding to this better differentiate the source freshness indicator from the source freshness anomalies one (but I think this is a given with this implementation).

Thanks for the great work once again
